unified_targets/configs/AIRF7.config

```diff
@@ -0,0 +1,87 @@
+board_name AIRF7
```

Comment:
This needs at least 

```
# Betaflight / STM32F7X2
```

as the first line of the file, or else it won't be recognised as a Unified Target configuration and configurator won't be able to select the correct Unified Target hex file to go with it.

Better yet, add the complete first line of the `diff` that this configuration was generated with - this will potentially help with debugging at a later stage.

Reply:
done